doc/source/getting_started/write_doc/guide_lines_tutorials.rst

```diff
@@ -0,0 +1,526 @@
+
+- Reference name;
+- Section title;
+- General description of the content of the tutorials in this section;
+- Cards with the tutorial title, description and applicable solvers (the card must have a link to the tutorial file);
+- Toctree with the tutorials in the section.
+
+.. code-block::
+
+    .. _ref_tutorial_new_section_template:
+
+    =============
+    Section title
+    =============
+
+    These tutorials demonstrate how to ...
+
+    .. grid:: 1 1 3 3
+        :gutter: 2
+        :padding: 2
+        :margin: 2
+
+        .. grid-item-card:: Tutorial title
+           :link: ref
+           :link-type: ref
+           :text-align: center
+
+           This tutorial ...
+
+           +++
+           :bdg-mapdl:`MAPDL` :bdg-lsdyna:`LS-DYNA` :bdg-fluent:`FLUENT` :bdg-cfx:`CFX`
+
+    .. toctree::
+        :maxdepth: 2
+        :hidden:
+
+        tutorial_file.rst
+
+You must also add the new section ``index.rst`` file in the main user guide page toctree. You can find this toctree
+at the end of ``doc/source/user_guide/index.rst`` file.
+
+.. code-block::
+
+    .. toctree::
+        :maxdepth: 2
+        :hidden:
+        :caption: Tutorials
+
+        tutorials/data_structures/index.rst
+        tutorials/language_and_usage/index.rst
+        tutorials/post_processing_basics/index.rst
+        tutorials/import_data/index.rst
+        tutorials/mesh/index.rst
+        tutorials/transform_data/index.rst
+        tutorials/export_data/index.rst
+        tutorials/plot/index.rst
+        tutorials/animate/index.rst
+        tutorials/enriching_dpf_capabilities/index.rst
+        tutorials/distributed_files/index.rst
+        tutorials/dpf_server/index.rst
+        tutorials/licensing/index.rst
+        tutorials/new_section/index.rst
+
+.. _ref_guide_lines_add_new_tutorial:
+
+=====================
+Adding a new tutorial
+=====================
+
+:download:`Download the tutorial card template<tutorial_card_template.rst>` :download:`Download the tutorial structure template<tutorial_structure_template.rst>`
+:download:`Download the tutorial content formating template<tutorial_content_template.rst>`
+
+Location and naming
+-------------------
+
+New tutorials must be added as ``.rst`` files to: ``doc/source/user_guide/tutorials/section_name/tutorial_file.rst``
+
+.. code-block::
+
+    .
+    ├── doc
+    │   ├── source
+    │   │    ├── user_guide
+    │   │    │   ├── tutorials
+    │   │    │        ├── section
+    │   │    │             ├── new_tutorial.rst
+
+You also have to add it to a card and the toctree on the tutorial section ``index.rst`` file. The card must have:
+
+- Tutorial title;
+- Short description;
+- Badges with the applicable solvers;
+- Link to the tutorial file;
+
+.. topic:: Card example
+
+    .. card:: Tutorial title
+       :text-align: center
+       :width: 25%
+
+       Short description of the tutorial
+
+       +++
+       :bdg-mapdl:`MAPDL` :bdg-lsdyna:`LS-DYNA` :bdg-fluent:`FLUENT` :bdg-cfx:`CFX`
```

Comment:
Same thing as for the section, could we directly do an `include` of the template rst?

Reply:
```
.. literalinclude:: tutorial_card_template.rst
   :language: rst
```